internal/sponge/sponge.go

```diff
@@ -0,0 +1,120 @@
+// Package sponge provides support for item importing.
+package sponge
+
+import (
+	"reflect"
+
+	"github.com/ardanlabs/kit/db"
+	"github.com/ardanlabs/kit/log"
+	"github.com/ardanlabs/kit/web/app"
+	"github.com/cayleygraph/cayley"
+	"github.com/coralproject/shelf/internal/sponge/item"
+	"github.com/coralproject/shelf/internal/wire"
+)
+
+// Import imports an item into the items collections and into the graph database.
+func Import(context interface{}, db *db.DB, graph *cayley.Handle, itm *item.Item) error {
+	log.Dev(context, "Import", "Started : ID[%s]", itm.ID)
+
+	// See if the item already exists.
```

Comment:
Change the comment to say See if the item does not exist.  This is because when this is the case the code inside the if is executed.

This code is an exception to rules (and it is valid) but it's not clear when it is executed as positive path.